### ISSUE TITLE
[UI] Completed questionnaires as webpage

### ISSUE THREAD

**Issue by kristenmcconnell** [department-of-veterans-affairs/va.gov-team#25081]
# Resources
- [Product Outline](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/checkin/product/product-outline.md)

# Description
To improve the accessibility of completed questionnaires, they should be available on a webpage as well as a PDF.

# Tasks
- [ ] Create completed questionnaire design in Sketch

# Acceptance Criteria
- [ ] Review with the team and accessibility
